doc/install.rst

```diff
@@ -88,7 +89,7 @@ PyGMT requires the following libraries:
 
 The following are optional (but recommended) dependencies:
 
-* `IPython <https://ipython.org/>`__: For embedding the figures in Jupyter
+* `JupyterLab <https://jupyter.org/>`__: For embedding the figures in Jupyter
   notebooks.
```

Comment:
I think we should still list "IPython" here because IPython is an optional dependency of PyGMT (PyGMT tries to import ipython), but JupyterLab isn't.

We will also list "ObsPy" here if we implement the ObsPy integration as in #967.

Reply:
Added in 4ea3c02.